commands.py

```diff
@@ -313,58 +416,85 @@ def error_not_on_channel(user: mantatail.UserConnection, channel_name: str) -> N
 def error_user_not_in_channel(
     user: mantatail.UserConnection, target_usr: mantatail.UserConnection, channel: mantatail.Channel
 ) -> None:
+    """
+    Sent when a user sends a channel-specific command with a user as an argument,
+    and this user is connected to the server but not to the channel.
```

Comment:
You connect to a server and join a channel. You don't connect to a channel.

```suggestion
    and this user is connected to the server but has not joined the channel.
```